src/vs/workbench/contrib/tasks/browser/task.contribution.ts

```diff
@@ -496,6 +496,12 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize('task.quickOpen.showAll', "Causes the Tasks: Run Task command to use the slower \"show all\" behavior instead of the faster two level picker where tasks are grouped by provider."),
 			default: false
 		},
+		'task.allowAutomaticTasks': {
+			type: 'boolean',
+			description: nls.localize('task.allowAutomaticTasks', "Allow Automatic Tasks in the folder"),
+			default: false,
+			restricted: true
+		},
```

Comment:
Are we doing the `'on' | 'auto' | 'off'` approach where on is always on, auto is prompt for every workspace and off is never prompt?

Reply:
I knew I was forgetting something 😅